website/content/en/pre-docs/deploy-guide/_index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Deployment Guide"
+linkTitle: "Deployment Guide"
+weight: 50
+---
+
+Introducing new software the modifies how Kubernetes operates is complex, and no single solution exists. 

```

Comment:
This is self evident from the remainder of this doc.
```suggestion
```

Reply:
This statement explicitly sets a mindset I want users to approach this doc with.